src/components/Header/elements.js

```diff
@@ -134,7 +93,40 @@ export const Overlay = styled.div`
   }
 `
 
-export const DesktopMenu = styled(Flex)`
+const DesktopMenuList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: ${remcalc(24)};
+  padding-right: ${remcalc(0)};
+  padding-top: ${remcalc(12)};
+  justify-content: center;
+
+  ${breakpoint('phone')`
+    padding-bottom: ${remcalc(24)};
+  `}
+
+  ${breakpoint('tablet')`
+    width: auto;
+    height: auto;
+    flex-direction: row;
+    position: relative;
+
+    > li:not(:last-child) {
+      margin-right: ${remcalc(18)};
+    }
+  `};
+
+  @media screen and (max-width: 768px) and (min-width: 600px) {
```

Comment:
again, if you see the diff, I just moved stuff around. I didn't want to rewrite the whole top nav, just improve some semantics and add the new component, so my main concern was to keep everything working as it was.

Reply:
Sure, but you're also creating 2 new components with the name `DesktopXYZ`. That's the only thing I'm really commenting on here.

Reply:
Not exactly, you're commenting on the styles — regarding why there's a media query for mobile when this is a 'desktop' named component.

Reply:
Sorry, this is just a miscommunication then. I was only intending to show, based on this line, why I found the naming confusing.

I can mark this resolved, since I've said the same thing in the other comment above